util/full-dependency-mapping.txt

```diff
@@ -0,0 +1,9 @@
+datetime-fortran libaccessom2 cice5 mom5 access-om2
+oasis3-mct libaccessom2 cice5 mom5 access-om2
+libaccessom2 cice5 mom5 access-om2
+cice5 access-om2
+mom5 access-om2
+access-om2 
+access-om3 
+fcm
+gcom
```

Comment:
How are these updated? Again just for interest!

Reply:
For now, they aren't updated. I did the dependent graph manually, through investigating the spack environment. It'd have to be updated manually if the dependents change. Issue raised here: https://github.com/ACCESS-NRI/build-ci/issues/21